### PR TITLE
Improve doc

### PR DESCRIPTION
**Pull request by mbaudin47**
Improve various help pages and examples.

### CODE REVIEW

**Comment by jschueller** [comment]
maybe include #2645 too

**Comment by mbaudin47** [comment]
> maybe include #2645 too

Ok, I agree. Done at 0bef9008f

File: python/src/GeneralLinearModelAlgorithm_doc.i.in [hunk 1]
```diff
@@ -9,24 +9,29 @@ Available constructors:
...
 
 Notes
 -----
-We suppose we have a sample :math:`(\vect{x}_k, \vect{y}_k)_{1 \leq k \leq N}` where :math:`\vect{y}_k = \cM(\vect{x}_k)` for all :math:`k`, with :math:`\cM:\Rset^n \mapsto \Rset^d` a given function.
-
-The objective is to build a metamodel :math:`\tilde{\cM}`, using a **general linear model**: the sample :math:`(\vect{y}_k)_{1 \leq k \leq N}` is considered as the restriction of a Gaussian process :math:`\vect{Y}(\omega, \vect{x})` on :math:`(\vect{x}_k)_{1 \leq k \leq N}`. The Gaussian process :math:`\vect{Y}(\omega, \vect{x})` is defined by:
+We suppose we have a sample :math:`(\vect{x}_k, \vect{y}_k)_{1 \leq k \leq \sampleSize}` where
+:math:`\sampleSize \in \Nset` is the sample size, :math:`\vect{y}_k = \model(\vect{x}_k)` for all :math:`k`
+and :math:`\model:\Rset^\inputDim \mapsto \Rset^\outputDim` is the model.
+The objective is to build a metamodel :math:`\metaModel`, using a **general linear model**: the sample
```
Comment (josephmure) [inline python/src/GeneralLinearModelAlgorithm_doc.i.in:31]:
```suggestion
The objective is to build a metamodel :math:`\metaModel`: the sample
```

File: python/src/GeneralLinearModelAlgorithm_doc.i.in [hunk 2]
```diff
@@ -9,24 +9,29 @@ Available constructors:
...
+:math:`(\vect{y}_k)_{1 \leq k \leq \sampleSize}` is considered as the restriction of a Gaussian process
```
Comment (josephmure) [inline python/src/GeneralLinearModelAlgorithm_doc.i.in:32]:
```suggestion
:math:`(\vect{y}_k)_{1 \leq k \leq \sampleSize}` is viewed as the restriction of a realization of a Gaussian process
```

File: python/src/GeneralLinearModelAlgorithm_doc.i.in [hunk 3]
```diff
@@ -55,23 +62,25 @@ We note:
...
 
 
-The *GeneralLinearModelAlgorithm* class estimates the coefficients :math:`\beta_j^\ell` and :math:`\vect{p}` where :math:`\vect{p}` is the vector of parameters of the covariance model (a subset of :math:`\vect{\theta}, \vect{\sigma}, \mat{R}, \vect{\lambda}`) that has been declared as *active* (by default, the full vectors :math:`\vect{\theta}` and :math:`\vect{\sigma}`).
-
-The estimation is done by maximizing the *reduced* log-likelihood of the model, see its expression below.
+The *GeneralLinearModelAlgorithm* class estimates the coefficients :math:`\beta_j^\ell` and :math:`\vect{p}`
+where :math:`\vect{p}` is the vector of parameters of the covariance model (a subset of
+:math:`\vect{\theta}, \vect{\sigma}, \mat{R}, \vect{\lambda}`) that has been declared as
+*active* (by default, the full vectors :math:`\vect{\theta}` and :math:`\vect{\sigma}`).
+The estimation is done by maximizing the *reduced* log-likelihood of the model (see its expression below).
```
Comment (josephmure) [inline python/src/GeneralLinearModelAlgorithm_doc.i.in:83]: Where is below? Maybe give the equation number?

**Review by josephmure — CHANGES_REQUESTED** [review]

**Review by adutfoy — APPROVED** [review]

File: python/src/GeneralLinearModelAlgorithm_doc.i.in (hunk 3 above)
Comment (mbaudin47) [reply python/src/GeneralLinearModelAlgorithm_doc.i.in]: Fixed at 2484465a1a62fa5784f2e76ce1411e17f892bf0e

**Review by josephmure — APPROVED** [review]
Once the numpy issue on OSX is solved this will be good to go IMO.

**Comment by mbaudin47** [comment]
> Once the numpy issue on OSX is solved this will be good to go IMO.

It's there: af4f2802767a6c4882f39a78275e87c5c7d0e36c

File: python/src/GeneralLinearModelAlgorithm_doc.i.in [hunk 4]
```diff
@@ -9,24 +9,31 @@ Available constructors:
 Parameters
 ----------
 inputSample, outputSample : :class:`~openturns.Sample` or 2d-array
```
Comment (josephmure) [inline python/src/GeneralLinearModelAlgorithm_doc.i.in:11]:
```suggestion
inputSample, outputSample : 2-d sequence of float
```

**Review by josephmure — APPROVED** [review]

**Comment by josephmure** [comment]
The failures are due to CircleCI issues independent of this PR, so I think it could be merged now.

**Comment by mbaudin47** [comment]
@jschueller : This could be merged in my opinion.

**Comment by jschueller** [comment]
there are conflicts, please rebase

File: python/src/CalibrationResult_doc.i.in [hunk 5]
```diff
@@ -65,6 +65,26 @@ parameterPrior : : :class:`~openturns.Distribution`
...
+| :class:`~openturns.LinearLeastSquaresCalibration`    | Distribution of the parameters with respect to randomness in the data |
++------------------------------------------------------+-----------------------------------------------------------------------+
+| :class:`~openturns.NonLinearLeastSquaresCalibration` | Distribution of the parameters with respect to randomness in the data |
++------------------------------------------------------+-----------------------------------------------------------------------+
+| :class:`~openturns.GaussianLinearCalibration`        | Posterior distribution of the parameters given the observations       |
++------------------------------------------------------+-----------------------------------------------------------------------+
+| :class:`~openturns.GaussianNonLinearCalibration`     | Distribution of the MAP with respect to randomness in the data        |
++------------------------------------------------------+-----------------------------------------------------------------------+
+
+**Table 1.** Content of the distribution returned by  `getParameterPosterior()` different classes returning a `CalibrationResult`.
```
Comment (jschueller) [inline python/src/CalibrationResult_doc.i.in:85]: maybe :meth: & :class: tags are missing here

Comment (mbaudin47) [reply python/src/CalibrationResult_doc.i.in]: You are right. Fixed at 44fd1989c0443d9213b781ed886bac00d58383a2

**Pull request merged**